ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -5,7 +5,7 @@ import {
   FormikStateLogger,
   EDTFSingleDatePicker,
 } from "@js/oarepo_ui";
-import { LocalVocabularySelectField } from "@js/oarepo_vocabularies";
+import { LocalVocabularySelectField, VocabularyTreeSelectField } from "@js/oarepo_vocabularies";
```

Comment:
Remove unused import local vocab select

Reply:
its used in languages and access rights because that's not a hierarchical data